Documentation/signing-and-verification-guide.md

```diff
@@ -0,0 +1,265 @@
+# Signing and Verification Guide
+
+This guide will walk you through signing, distributing, and verifying the hello ACI created in the [getting started guide](getting-started-guide.md).
+
+```
+hello-0.0.1-linux-amd64.aci
+```
+
+* [Signing ACI images](#signing-aci-images)
+* [Distributing Images via Meta Discovery](#distributing-images-via-meta-discovery)
+* [Verifying Images with Rocket](#verifying-images-with-rocket)
+  * [Establishing Trust](#establishing-trust)
+  * [Example Usage](#example-usage)
+
+## Signing ACI Images
+
+### Generate a gpg signing key
```

Comment:
I would like to see it called out explicitly when/why this is necessary, e.g. "if you already have a gpg key, ..."